resources/benchmark-runner.mjs

```diff
@@ -163,6 +163,166 @@ const WarmupSuite = {
     ],
 };
 
+class TimerCallbacks {
+    constructor(firstCallback, secondCallback) {
```

Comment:
Call these as first & second are rather confusing.
We should probably call them as syncCallback & asyncCallback.